pkg/rkecerts/expiration.go

```diff
@@ -0,0 +1,59 @@
+package rkecerts
+
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+	"time"
+
+	rkeCluster "github.com/rancher/rke/cluster"
+	"github.com/rancher/rke/pki"
+	"github.com/rancher/rke/pki/cert"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	v1 "k8s.io/api/core/v1"
+)
+
+func CleanCertificateBundle(certs map[string]pki.CertificatePKI) {
+	for name := range certs {
+		if strings.Contains(name, "client") ||
```

Comment:
nit: little hard to read what this is doing, maybe put the boolean checks in another func

Reply:
this actually was another bug, basically api-proxy-client is a legit cert and shouldn't be skipped from the check

Reply:
removed client from check